crates/db_views/src/comment_view.rs

```diff
@@ -160,7 +163,7 @@ fn queries<'a>() -> Queries<
         creator_is_moderator,
         creator_is_admin,
         subscribed_type_selection,
-        is_saved_selection,
+        is_saved_selection.is_not_null(),
```

Comment:
We don't need to add any new fields to the view, since we can just check the existence of that join.